web/src/views/SubmissionPortal/Components/SubmissionList.vue

```diff
@@ -99,81 +100,107 @@ export default defineComponent({
 </script>
 
 <template>
-  <v-card flat>
-    <v-card-text class="pt-0 px-0">
-      <v-container>
-        <v-row>
-          <v-col class="pb-0">
-            <TitleBanner />
-            <IconBar />
-          </v-col>
-        </v-row>
-        <v-row v-if="submission.data.results.count === 0">
-          <v-col>
-            <IntroBlurb />
-          </v-col>
-        </v-row>
-      </v-container>
-    </v-card-text>
-    <v-card-text>
-      <v-btn
-        color="primary"
-        @click="createNewSubmission"
-      >
-        <v-icon>mdi-plus</v-icon>
-        Create New Submission
-      </v-btn>
-    </v-card-text>
-    <v-card-title class="text-h4">
-      Past submissions
-    </v-card-title>
-    <v-card-text>
-      Pick up where you left off or review a previous submission.
-    </v-card-text>
-    <v-card outlined>
-      <v-data-table
-        :headers="headers"
-        :items="submission.data.results.results"
-        :server-items-length="submission.data.results.count"
-        :options.sync="options"
-        :loading="submission.loading.value"
-        :items-per-page.sync="submission.data.limit"
-        :footer-props="{ itemsPerPageOptions: [10, 20, 50] }"
-      >
-        <template #[`item.author.name`]="{ item }">
-          <orcid-id
-            :orcid-id="item.author.orcid"
-            :name="item.author.name"
-            :width="14"
-            :authenticated="true"
-          />
-        </template>
-        <template #[`item.metadata_submission.templates`]="{ item }">
-          {{ item.metadata_submission.templates.map((template) => HARMONIZER_TEMPLATES[template].displayName).join(' + ') }}
-        </template>
-        <template #[`item.created`]="{ item }">
-          {{ new Date(item.created).toLocaleString() }}
-        </template>
-        <template #[`item.status`]="{ item }">
-          <v-chip
-            :color="getStatus(item).color"
+  <div>
+    <v-menu
+      offset-x
+      left
+    >
+      <template #activator="{on, attrs}">
+        <v-btn
+          color="primary"
+          large
+          class="mr-0"
+          style="transform:translateY(-50%) rotate(-90deg); right: -50px; top: 50%; position: absolute; z-index: 100;"
+          v-bind="attrs"
+          v-on="on"
+        >
+          support
+          <v-icon
+            class="ml-2"
+            style="transform: rotate(90deg);"
           >
-            {{ getStatus(item).text }}
-          </v-chip>
-        </template>
-        <template #[`item.action`]="{ item }">
-          <v-btn
-            small
-            color="primary"
-            @click="() => resume(item)"
-          >
-            Resume
-            <v-icon class="pl-1">
-              mdi-arrow-right-circle
-            </v-icon>
-          </v-btn>
-        </template>
-      </v-data-table>
+            mdi-message-question
+          </v-icon>
+        </v-btn>
+      </template>
+      <ContactCard />
```

Comment:
Would it make sense to have the contact card be sticky? Currently it moves around as I scroll around the page. In extreme scenarios (changing the page size to be 50), I can scroll so it is off the screen. 

https://github.com/microbiomedata/nmdc-server/assets/7085625/f999b6a5-d75f-4c91-942b-0390e1c55c6b

Reply:
fixed in cad8fa53883b8dff5e229e07cdf7734676eaa46b